config/locales/forms/en.yml

```diff
@@ -17,7 +17,7 @@ en:
       show_hdr: Create a strong password
     messages:
       current_address: You should be be able to receive mail at this address.
-      remember_device: Remember this device for %{duration} days
+      remember_device: Remember this browser for %{duration} days
```

Comment:
Does it make sense to day "Remember this browser"? That makes it sound like we are not requiring their username / password next time. Maybe something along the lines of "Remember this phone on this browser" would make more sense?

Reply:
Yeah, let's keep the discussion in that ticket.